src/index.js

```diff
@@ -743,6 +742,7 @@ function createConfig(options, entry, format, writeMeta) {
 
 		/** @type {import('rollup').OutputOptions} */
 		outputOptions: {
+			exports: 'named',
```

Comment:
Does this break default exports?

Reply:
Unsure yet, I was just removing warnings 😛